.gitignore

```diff
@@ -30,3 +30,8 @@ tmp/**/*
 .idea/*.xml
 .sass-cache
 nbproject
+
+/vendor/bundle
+/spec/dummy_app/vendor/bundle
+*.iml
```

Comment:
we always add this to every gitignore file in every project, but actually it also make sense to add this to global gitignore file, of course this would require every developer to stick to some rules and etc. and etc. , but in this very sense it make sense to enforce this :smile: 

the way I do it:

add to `~/.gitconfig` this section

```bash
[core]
  excludesfile = ~/.gitignore
```

and then in `~/.gitignore` add anything you like from this project https://github.com/github/gitignore/tree/master/Global , for example Intellij Idea > 15

```bash
# Intellij IDEA >15
*.iml
```